index.html

```diff
@@ -16,7 +16,7 @@
         <li><a href="#intro">Welcome</a></li>
         <li><a href="#join-the-fight">Join the Fight</a></li>
         <li>
-          <a href="#rules-submission-faq">Rules, Submissions &amp; FAQ</a>
```

Comment:
Oh!  Maybe not do this, it breaks the scrolling effect on the page.  I was picturing having a very tiny section in the main page that then has a link to the separate page.  Then, we don't break the scroll effect.

Reply:
The scrolling effect looks to be working for me, as I scroll by it just skips that section and jumps to "Links & Resources". Was there another issue you're seeing?

Can we table this for now, and I'll create a separate story to look at changing the nav? We're already doing so much work on the homepage that I'd like to wait and see how Marjorie, et al want to proceed with that work before introducing another nav element on the page somewhere else.

Reply:
Oh, then never mind.  Maybe it's external links that break it.  Your changes are good then!